pkg/platform/telemetry/telemetry.go

```diff
@@ -68,19 +68,23 @@ func (ac *Telemetry) Ping(isTestMode bool) {
 			}
 
 			if count == 0 {
-				bin, err := marshalEvent(event, ac.logger)
-				if err != nil {
-					break
-				}
-				resp, err := http.Post("https://telemetry.keploy.io/analytics", "application/json", bytes.NewBuffer(bin))
-				if err != nil {
-					ac.logger.Debug("failed to send request for analytics", zap.Error(err))
-					break
-				}
-
-				id, err := unmarshalResp(resp, ac.logger)
-				if err != nil {
-					break
+				id, _ := ac.store.Get(false)
```

Comment:
When Keploy binary will run with test_export as false, then it should not check for installation id in the file.

Reply:
Added the changes